CONTRIBUTING.md

```diff
@@ -214,40 +214,40 @@ Don't put docs in .h files, if there is a corresponding .cpp file.
 #include "blabla.h"
 
 /**
-I can be briefly described as well!
-*/
+ * I can be briefly described as well!
```

Comment:
Function statements can have `@brief` tags infront of the first line of descriptions too. Also perhaps demonstrate a function with parameters and a return statement, e.g:

```c++
/**
 * @brief I can be briefly described as well!
 *
 * And here goes my longer descrption!
 *
 * @param x Description for the first parameter
 * @param y Description for the second paramater
 * @return An amazing result
 */
static int example(int x, int y)
{
    // Function implementation...
}
```